apimonitor/tests/test_health.py

```
@@ -0,0 +1,191 @@
+            mock_response.status_code = 200
+            
+            mock_client = Mock()
+            mock_client.request.return_value = mock_response
+            mock_client.__enter__.return_value = mock_client
+            mock_client.__exit__.return_value = None
+            
+            mock_client_class.return_value = mock_client
+            
+            call_health(service)
+        
+        assert result['status_code'] == 200
```

Ruff (0.12.2)

Undefined name `result`

(F821)